main.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 const (
-	jql = `project in (ROX)
+	jql = `project in (%q)
```

Comment:
we do not have quotes before but I've checked it and it's working at least in UI

Reply:
I'll change to a %s.